scalafix-tests/input/src/main/scala/test/DisableSyntaxMoreRules.scala

```diff
@@ -53,6 +53,12 @@ Default args makes it hard to use methods as functions.
     def foobarfoo(x: Int = 42) = 6 // assert: DisableSyntax.defaultArgs
   }
 
+  class Bar { type Foo = Int; def foo = 42 }
+  def foo(a: { type Foo = Int; def foo: Foo } = new Bar): Int = a.foo /* assert: DisableSyntax.defaultArgs
```

Comment:
```scala
def foo(a: { type Foo = Int; def foo: Foo } = new Bar): Int = a.foo
                                              ^ (param.default)
                                            ^ (token)
```

Reply:
There is no benefit to pointing the caret at `=` over `new`, and with range positions it will be way clearer if `new Bar` is highlighted instead of only `=`